composer.json

```diff
@@ -16,27 +17,36 @@
         "spryker/glossary": "^3.0.0",
         "spryker/gui": "^3.0.0",
         "spryker/kernel": "^3.33.0",
+        "spryker/messenger": "^3.1.0",
         "spryker/money": "^2.0.0",
-        "spryker/payment": "^4.0.0 || ^5.0.0",
+        "spryker/payment": "^4.3.0 || ^5.0.0",
         "spryker/propel-orm": "^1.0.0",
-        "spryker/quote": "^2.0.0",
+        "spryker/quote": "^2.14.3",
         "spryker/refund": "^5.0.0",
         "spryker/sales": "^8.0.0 || ^10.0.0 || ^11.0.0",
         "spryker/shipment": "^6.0.0 || ^7.0.0 || ^8.0.0",
         "spryker/step-engine": "^3.1.0",
-        "spryker/symfony": "^3.0.0",
+        "spryker/symfony": "^3.4.0",
         "spryker/transfer": "^3.25.0",
         "spryker/util-encoding": "^2.0.0",
         "spryker/util-text": "^1.1.0"
     },
     "require-dev": {
+        "codeception/module-asserts": "^1.3",
+        "codeception/module-webdriver": "^1.4",
+        "phpstan/phpstan": "*",
         "spryker-shop/shop-application": "^1.0.0",
-        "spryker/application": "^3.0.0",
```

Comment:
This has really no side-effects?

Reply:
Yes, it was moved to "require" section.

Reply:
Regarding to all changes with packages in composer.json. If I revert them, we have the first problem like this:

$ vendor/bin/console dev:composer:update-json-files -d -v
Store: DE | Environment: ci.pgsql
1 of 1034 module(s) need(s) updating.
SprykerEco.Braintree
Please run this command locally without dry-run:
vendor/bin/console dev:composer:update-json-files SprykerEco.Braintree
The command "vendor/bin/console dev:composer:update-json-files -d -v" exited with 1.

As I said previously, the part of my changes in composer.json were made automatically by execution of this command:
console dev:composer:update-json-files -d -v

The next one: 
$ vendor/bin/console dev:dependency:find
Store: DE | Environment: ci.pgsql
+-------------------------+----------------------------------------------------+
| Dependency violations in SprykerEco.Braintree                                |
+-------------------------+----------------------------------------------------+
| spryker/container       | Test dependency not listed in composer require-dev |
| spryker/error-handler   | Test dependency not listed in composer require-dev |
| spryker/event           | Test dependency not listed in composer require-dev |
| spryker/log             | Test dependency not listed in composer require-dev |
| spryker/messenger       | Source dependency not listed in composer require   |
| spryker/monolog         | Test dependency not listed in composer require-dev |
| spryker/queue           | Test dependency not listed in composer require-dev |
+-------------------------+----------------------------------------------------+
7 module dependency issues found
The command "vendor/bin/console dev:dependency:find" exited with 1.

Reply:
All issues were fixed again.